minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java

```diff
@@ -128,6 +130,21 @@ public void initConfiguration() throws IOException {
             mHostname, mWorkDirectory).entrySet()) {
       ServerConfiguration.set(entry.getKey(), entry.getValue());
     }
+    if (mJournalType == JournalType.UFS) {
+      ServerConfiguration.set(PropertyKey.MASTER_JOURNAL_TYPE, JournalType.UFS);
+    } else {
+      ServerConfiguration.set(PropertyKey.MASTER_JOURNAL_TYPE, JournalType.EMBEDDED);
+      ServerConfiguration.set(PropertyKey.MASTER_EMBEDDED_JOURNAL_ADDRESSES,
+          String.format("%s:%s", mHostname,
+              ServerConfiguration.get(PropertyKey.MASTER_EMBEDDED_JOURNAL_PORT)));
+      ServerConfiguration.set(PropertyKey.MASTER_RPC_ADDRESSES, String.format("%s:%s", mHostname,
+          ServerConfiguration.get(PropertyKey.MASTER_RPC_PORT)));
+      if (ServerConfiguration.getBytes(PropertyKey.MASTER_JOURNAL_LOG_SIZE_BYTES_MAX) < 65) {
```

Comment:
it's not good to silently change the configuration when other code explicitly tries to set the configuration to something illegal. This could be confusing to the person trying to set the log size to 64. We should allow it to fail so that we can fix their code

Reply:
I changed to set the `MASTER_JOURNAL_LOG_SIZE` to its default value in `ConfigurationTestUtils.testConfigurationDefaults`. Users can set explicit values to this property in their test and if the value is smaller than 65, the person will notice that the test fail and find out the reason.